.github/workflows/rundoc-pr.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           delete-branch: true
           base: main
           commit-message: Update tutorials
-          title: Update tutorials
+          title: Update ${{matrix.lang}} tutorial
```

Comment:
(not blocking)

When reviewing the prior PR I hadn't spotted that each language would open a separate PR. With the non-determinism this will mean 8 PRs every week, even if there haven't been any changes. I think we either need to fix the non-determinism regressions, or else switch to having a single PR (which will at least mean it's 1x guaranteed PR instead of 8x PRs).

Reply:
https://github.com/heroku/buildpacks/pull/75/commits/89713eb2362eef2ed76c12f6e5965b2f482d5870